### PR TITLE
remove viewer type and update tests

### PR DESCRIPTION
**Pull request by ssomnoremac**
Taking a stab at this. I'm not %100 sure what I'm doing as I'm using a different language for my own graphQL server but I was able to get tests to pass.

### CODE REVIEW

File: src/loader/UserLoader.js [hunk 1]
```diff
@@ -37,12 +37,11 @@ export default class User {
 
 export const getLoader = () => new DataLoader(ids => mongooseLoader(UserModel, ids));
 
-const viewerCanSee = (viewer, data) => {
+const meCanSee = (me, data) => {
```
Comment (sibelius) [inline src/loader/UserLoader.js]: viewerCanSee continues the same

Comment (sibelius) [inline src/loader/UserLoader.js]: Viewer is like me, who is viewing the schema

**Comment by ssomnoremac** [comment]
Fix for #46

File: src/loader/UserLoader.js (hunk 1 above)
Comment (ssomnoremac) [inline src/loader/UserLoader.js]: ok, just wasn't sure about keeping the "viewer" terminology. I had `meCanSee` at one point 😄

Comment (ronal2do) [inline src/loader/UserLoader.js]: Greater work

Comment (ssomnoremac) [inline src/loader/UserLoader.js]: oh, wait I see what you're saying, nevermind.

Comment (sibelius) [inline src/loader/UserLoader.js]: no problem, we can rethink this in another issue or PR

**Comment by sibelius** [comment]
awesome work 💯 

this should be a medium post

`How to remove ViewerType from your GraphQL server`

**Pull request merged**